THIRD_PARTY_LICENSES

```diff
@@ -344,7 +348,9 @@ Copyright (c) .NET Foundation and Contributors
 Copyright (c) .NET Foundation and Contributors
 ** Microsoft.OpenApi; version 1.2.3 -- https://www.nuget.org/packages/Microsoft.OpenApi/
 Copyright (c) Microsoft Corporation.
-
+** System.Text.Json; version 6.0.4 -- https://www.nuget.org/packages/System.Text.Json
```

Comment:
This moved when I regenerated the whole file from the internal attribution tool.